Library/Homebrew/cmd/audit.rb

```diff
@@ -367,6 +367,10 @@ def audit_desc
     if desc =~ %r[[Cc]ommand line]
       problem "It should be \"command-line\", not \"command line\"."
     end
+
+    if desc =~ %r[^([Aa]n?)\s]
+      problem "Description begins with indefinite article: \"#{$1}\""
```

Comment:
@xu-cheng I was uncertain which wording to use and will happily change it.
 
@mikemcquaid That's what the `#{$1}` is for, so people know which indefinite article was used in the description.